3.2/s2i/bin/usage

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+man -l /help.1
```

Comment:
+1 for using groff instead of man. However we should use `groff -t -man -ETascii /help.1` to properly format any tables located in the help file.